app/controllers/admin/instances_controller.rb

```diff
@@ -1,5 +1,88 @@
 class Admin::InstancesController < AdminController
+  skip_before_action :verify_authenticity_token
+
   def index
+    @instances = {}
+
+    respond_to do |format|
+      format.html
+      format.json { render json: @instances }
+    end
+  end
+
+  def edit
+    # -
+  end
+
+  def access
+    # -
+  end
+
+  def collaborators
+    # -
+  end
+
+  def stats
     # -
   end
+
+  def logs
+    # -
+  end
+
+  def deploy
+    sleep(5)
+
+    @status = {
+      level: 'success',
+      message: 'online',
+      icon: 'fa fa-world'
```

Comment:
what is this icon ? should not be in the controller

Reply:
I'll move that to the JS component. It's the icon to the left of the status label text.